components/library/libjpeg-turbo/libjpeg-turbo.p5m

```diff
@@ -26,9 +27,6 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 # Require package with default JPEG API
 depend fmri=image/library/libjpeg8-turbo type=require
```

Comment:
You do not touch the `Makefile` for this component.  Will it be built?

Reply:
Good catch, this got lost during the rebase.